### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by Superpat** [LesFruitsDefendus/saskatoon-ng#641]
We need to go over the browser accessibility checks. For instance there are many WCAG violations around text contrast for some ui elements.
